Makefile

```diff
@@ -45,8 +45,6 @@ default-config:
 js:
 	docker run --rm $(DOCKER_FLAGS) --user $(current_user):$(current_group) -v $(BUILD_DIR):/data:delegated -w /data -e NO_UPDATE_NOTIFIER=1 -e REDUX_LOG=$(REDUX_LOG) node:8 npm run build-assets
 	docker run --rm $(DOCKER_FLAGS) --user $(current_user):$(current_group) -v $(BUILD_DIR):/data:delegated -w /data -e NO_UPDATE_NOTIFIER=1 node:8 npm run copy-assets
-	#Ignore TypeScript errors on build until resolved
-	-docker run --rm $(DOCKER_FLAGS) --user $(current_user):$(current_group) -v $(BUILD_DIR):/data:delegated -w /data/skins/cat17 -e NO_UPDATE_NOTIFIER=1 node:8 npm run build-address-form
```

Comment:
Because `build-address-form` is now part of the `build-assets` tasks, which enables parallel processing of all the asset building on multi-cpu machines, speeding up the build process.